Patch origin: in-house
Patch status: Solaris-specific; not suitable for upstream

Need to use an appropriate path on Solaris.

--- openca-ocspd-3.1.2/scripts/ocspd-genreq.sh.in	2015-11-10 13:31:41.108065796 -0800
+++ openca-ocspd-3.1.2/scripts/ocspd-genreq.sh.in	2016-04-08 12:10:37.931458124 -0700
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-prefix="@prefix@"
+prefix=
 token="ocspServerToken"
 defSubject="CN=OCSP Server, O=OpenCA, C=US"
 defAlgor="RSA-SHA256"
